src/goto-instrument/contracts/contracts.cpp

```diff
@@ -441,16 +452,14 @@ void code_contractst::check_apply_loop_contracts(
   goto_function.body.destructive_insert(
     loop_head, add_pragma_disable_assigns_check(pre_loop_head_instrs));
 
-  optionalt<cfg_infot> cfg_empty_info;
+  // HERE
+  //  loop_cfg_infot cfg_info(goto_function, loop);
```

Comment:
I am unclear on this. This is probably a stale comment?
If line 456 is still relevant, could you please expand the comment on line 455?

Reply:
this is a leftover comment